ee-9/feature-pack/pom.xml

```diff
@@ -544,6 +544,9 @@
                                 <exclude>org.wildfly:wildfly-clustering-infinispan-extension\z</exclude>
                                 <exclude>org.wildfly:wildfly-clustering-web-infinispan\z</exclude>
 
+                                <!-- Contains module names which have been aliased, additional transformation not required. -->
+                                <exclude>org.wildfly:wildfly-microprofile-jwt-smallrye\z</exclude>
```

Comment:
Should not this module be excluded from the maven scope instead of excluding the transformation?

I mean, exclude it here: https://github.com/wildfly/wildfly/blob/8fc816d39a3c22fb6b684b79f89e4d3449a3ddd9/ee-9/feature-pack/pom.xml#L668, because wildfly-microprofile-jwt-smallrye-jakarta is its replacement.

Reply:
Good point!  Thanks @yersan .

Reply:
Thanks @yersan I will add that exclude and remove this one.